whois-update/src/main/java/net/ripe/db/whois/update/mail/MailGatewaySmtp.java

```diff
@@ -82,22 +80,24 @@ public void sendEmail(final String to, final String subject, final String text,
     @RetryFor(value = MailSendException.class, attempts = 20, intervalMs = 10000)
     private void sendEmailAttempt(final String to, final String replyTo, final String subject, final String text) {
         try {
-            mailSender.send(new MimeMessagePreparator() {
-                @Override
-                public void prepare(final MimeMessage mimeMessage) throws MessagingException {
-                    final MimeMessageHelper message = new MimeMessageHelper(mimeMessage, MimeMessageHelper.MULTIPART_MODE_NO, "UTF-8");
-                    message.setFrom(mailConfiguration.getFrom());
-                    message.setTo(to);
-                    if (!StringUtils.isEmpty(replyTo))
-                        message.setReplyTo(replyTo);
-                    message.setSubject(subject);
-                    message.setText(text);
-
-                    mimeMessage.addHeader("Precedence", "bulk");
-                    mimeMessage.addHeader("Auto-Submitted", "auto-generated");
-
-                    loggerContext.log("msg-out.txt", new MailMessageLogCallback(mimeMessage));
+            final String punyCodedTo = PunycodeConversion.toAscii(to);
```

Comment:
Why are these lines outside the send() method?

Reply:
Because I started off using the existing mocking setup type tests with argument captures but injecting the email did not work.. now I'm using the MailSenderStub and it works just fine so I've moved them into the send(), well spotted :)